lib/service/cfg.go

```diff
@@ -98,6 +98,11 @@ type Config struct {
 	// misleading.
 	AuthServers []utils.NetAddr
 
+	// AuthServers is a list of auth servers, proxies and peer auth servers to
```

Comment:
```suggestion
	// ProxyServers is a list of auth servers, proxies and peer auth servers to
```

Also, looks like this is just a copy paste comment.

I would update the commend on AuthServers to mention that it should only be used for auth servers going forward, and then clarify that proxy servers should be used for tunneling.